src/main/java/school/lemon/changerequest/java/introduction/hw1/Task4.java

```diff
@@ -4,6 +4,8 @@
 public class Task4 {
 
     public static double getSumOfGreatest(double a, double b, double c) {
-        return 0;
+        double m1 = Math.max(a,b);
+        double m2 = Math.max(b,c);
+        return m1+m2;
```

Comment:
Really nice solution, no one have done it yet with Math.max.

Also try to do it without Math methods.